src/inboxsdk-js/inboxsdk-NONREMOTE.ts

```diff
@@ -1,8 +1,7 @@
-import shimLoaderLoadScript from './loading/load-platform-implementation-NONREMOTE';
-import loader from './loading/platform-implementation-loader';
-
-loader._loadScript = shimLoaderLoadScript(500);
-
+import { loadPi as shimLoaderLoadScript } from './loading/load-platform-implementation-NONREMOTE';
+import { PlatformImplementationLoader } from './loading/platform-implementation-loader';
 import * as SDK from './inboxsdk';
 
+PlatformImplementationLoader.loadScript = shimLoaderLoadScript(0);
```

Comment:
This should have a timeout.

Reply:
Having toyed with this a bit, it _does_ seem that the timeout is load bearing. A timeout of zero doesn't seem to reliably load the SDK in the toy extension at examples/thread-rows.